src/bundle/Resources/public/scss/_split-button.scss

```diff
@@ -0,0 +1,52 @@
+.ibexa-split-btn {
+    display: flex;
+
+    &__main-btn,
+    &__toggle-btn {
+        &:hover {
+            z-index: 1;
+        }
+    }
+
+    &__main-btn {
+        border-top-right-radius: 0px;
+        border-bottom-right-radius: 0px;
+    }
+
+    &__split {
+        display: none;
+        width: calculateRem(0.5px);
```

Comment:
Do we want to use half pixels? I think there were some problems between browsers/systems using it, but I'm not sure now

Reply:
AFAIR I've done this for the retina to look better and AFAIK we use half pixels.
Actually, it will be served to the browser in `rem` where we use non-integer values all the time.